gluon/html.py

```diff
@@ -186,6 +188,8 @@ def URL(
     within the present application and controller.
 
     Args:
+        language:
```

Comment:
Niphold please state what you want and I fix that...

On Tue, Mar 15, 2016 at 4:37 PM, niphlod <notifications@github.com> wrote:

> In gluon/html.py
> <https://github.com/web2py/web2py/pull/1219#discussion_r56236185>:
>
> > @@ -186,6 +188,8 @@ def URL(
> >      within the present application and controller.
> >
> >      Args:
> > +        language:
>
> here too . Another point for brains over IDEs with fancy plugins....
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/web2py/web2py/pull/1219/files/00632a9dec8ce371ea42b0a7ae2bdbe2d94aea4c#r56236185>
>